scripts/zowe-api-mediation-configure.sh

```diff
@@ -139,6 +142,15 @@ services:
           serviceRelativeUrl: api/v1/datasets
         - gatewayUrl: ui/v1  # [api/ui/ws]/v{majorVersion}
           serviceRelativeUrl: ui/v1/datasets
+      apiInfo:
+        - apiId: com.ibm.datasets
+          gatewayUrl: api/v1
+          version: 1.0.0
```

Comment:
Can you update the version to 0.9.3 (I know it doesn't match the v1 flag, but we know they will change before 1.0)